packages/cadl-ranch-specs/http/authentication/http/custom/main.tsp

```diff
@@ -0,0 +1,39 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+
+using TypeSpec.Http;
+
+@scenarioService("/authentication/http/custom")
+@doc("Illustrates clients generated with generic HTTP auth.")
+@useAuth({
+  type: AuthType.http,
+  scheme: "SharedAccessKey",
+})
```

Comment:
It seems hard to understand from tsp, that this is to create a `SharedAccessKey <value>` on `Authorization` header.
(maybe the `Authorization` is kind of inferred, as `http` type is mostly on it; but `scheme` as prefix is quite confusing to anyone out of codegen)